src/utils.ts

```diff
@@ -602,6 +602,27 @@ export const throttle = <T extends (...args: unknown[]) => unknown>(
   };
 };
 
+const get = <T>(obj: T, path: string): unknown =>
+  path.split('.').reduce<unknown>((acc, key) => {
+    if (acc && typeof acc === 'object' && key in acc) {
+      return (acc as Record<string, unknown>)[key];
+    }
+    return undefined;
+  }, obj);
+
+// works exactly the same as lodash.uniqBy
+export const uniqBy = <T>(array: T[] | unknown, iteratee: ((item: T) => unknown) | keyof T): T[] => {
+  if (!Array.isArray(array)) return [];
+
+  const seen = new Map<unknown, boolean>();
```

Comment:
nit-picking, but I think you can go with `new Set<unknown>()` here.

Reply:
Totally ! I think I started off with something else in mind and then just never changed it. Thanks !